holidayextras/brands
# FP-3342 amended booking flow navigation colour

JIRA: https://hxshortbreaks.atlassian.net/browse/FP-3342

#### What does this PR do? (please provide any background)
The JIRA was to change the styling from FrightNight back to ThorpeDark. During the change I noticed the booking flow navigation inactive text links were white and not visible on the white background. I have amended the colour to the brand colour as shown in the screenshot below.

#### How can this be tested?
`cd pattern-library && npm link ../brands/`

- link `@holidayextras/brand-thorpepark-dark` to `the wall` from `brands` 
- make  a change to the stylesheet in `the-wall` so the styles update
- Check that the booking flow navigation looks like in the screenshot below. It should NOT have white text for the INACTIVE links e.g. in the screenshot below Upgrades and Payment being inactive.

#### Any tech debt?
No

#### Screenshots / Screencast
![screen shot 2017-11-07 at 20 08 55](https://user-images.githubusercontent.com/32061182/32515821-37bb0354-c3f9-11e7-9a65-68f7d98b732e.png)


#### What gif best describes how you feel about this work?
![]()


---

**Reviewer**

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.
